websites/D/diziwatch/presence.ts

```diff
@@ -0,0 +1,81 @@
+const presence = new Presence({
+		clientId: "1214974615202299994",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://i.imgur.com/45R11f9.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: Assets.Logo,
```

Comment:
```suggestion
		largeImageKey: Assets.Logo,
		startTimestamp: browsingTimestamp,
```

Reply:
This has not been resolved.

Reply:
Simplify the code/remove duplication by placing the `startTimestamp` field in the initial data.